docs/intent.md

```diff
@@ -0,0 +1,29 @@
+# Intent
+
+Intent to change anything on an organization and its apps, through an Ethereum transaction. Allows to get transaction paths from it.
```

Comment:
Wonder if we should mention that an intent might also trigger an action on an external contract using Agent.

Reply:
Yes true!